[2022-10-27 08:43:09.527601] INFO: hello world 专用: 有人访问了
[2022-10-27 08:49:54.391881] INFO: hello world 专用: 有人访问了
[2022-10-27 08:49:56.631736] INFO: hello world 专用: 有人访问了
[2022-10-27 08:49:57.348350] INFO: hello world 专用: 有人访问了
[2022-10-27 16:58:14.687467] INFO: hello world 专用: 有人访问了
[2022-10-27 17:19:41.007958] INFO: hello world 专用: 有人访问了
[2022-10-27 17:20:51.709261] INFO: hello world 专用: 有人访问了
[2022-10-28 14:19:55.890067] ERROR: userDao: 用户注册失败:(_mysql_exceptions.OperationalError) (1267, "Illegal mix of collations (latin1_swedish_ci,IMPLICIT) and (utf8_general_ci,COERCIBLE) for operation '='")
[SQL: SELECT user.id AS user_id, user.username AS user_username, user.name AS user_name, user.password AS user_password, user.email AS user_email, user.`role` AS user_role, user.created_at AS user_created_at, user.updated_at AS user_updated_at, user.deleted_at AS user_deleted_at, user.last_login_at AS user_last_login_at 
FROM user 
WHERE user.username = %s OR user.email = %s OR user.name = %s OR user.password = %s]
[parameters: ('hanfengju', '17600007601@163.com', '韩凤举', 'hanfengju1992')]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
[2022-10-28 14:21:07.786465] ERROR: userDao: 用户注册失败:用户名或邮箱已存在
[2022-10-28 14:21:14.554682] ERROR: userDao: 用户注册失败:用户名或邮箱已存在
[2022-10-28 14:21:27.037285] ERROR: userDao: 用户注册失败:(_mysql_exceptions.OperationalError) (1267, "Illegal mix of collations (latin1_swedish_ci,IMPLICIT) and (utf8_general_ci,COERCIBLE) for operation '='")
[SQL: SELECT user.id AS user_id, user.username AS user_username, user.name AS user_name, user.password AS user_password, user.email AS user_email, user.`role` AS user_role, user.created_at AS user_created_at, user.updated_at AS user_updated_at, user.deleted_at AS user_deleted_at, user.last_login_at AS user_last_login_at 
FROM user 
WHERE user.username = %s OR user.email = %s OR user.name = %s OR user.password = %s]
[parameters: ('hanfengju', '17600007601@163.com', '韩凤举', 'hanfengju1992')]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
[2022-10-28 14:21:38.705439] ERROR: userDao: 用户注册失败:用户名或邮箱已存在
[2022-10-28 14:23:24.894599] ERROR: userDao: 用户注册失败:用户名或邮箱已存在
[2022-10-28 14:24:04.745965] ERROR: userDao: 用户注册失败:用户名或邮箱已存在
[2022-10-28 14:24:55.813952] ERROR: userDao: 用户注册失败:用户名或邮箱已存在
[2022-10-28 14:25:18.536397] ERROR: userDao: 用户注册失败:Unicode-objects must be encoded before hashing
[2022-10-28 14:27:23.092686] ERROR: userDao: 用户注册失败:用户名或邮箱已存在
[2022-10-28 14:45:30.145620] ERROR: userDao: 用户注册失败:用户名或邮箱已存在
[2022-10-28 14:53:00.409976] ERROR: userDao: 用户注册失败:用户名或邮箱已存在
[2022-10-28 14:57:16.107894] ERROR: userDao: 用户注册失败:用户名或邮箱已存在
[2022-10-28 15:05:21.586074] ERROR: userDao: 用户注册失败:用户名或邮箱已存在
[2022-10-28 16:23:10.514078] ERROR: userDao: 用户hanfengju登录失败:filter() got an unexpected keyword argument 'username'
[2022-10-28 18:04:00.918754] INFO: userDao: 有人访问了
[2022-10-31 07:16:20.349339] INFO: userDao: 有人访问了
[2022-10-31 07:16:25.450956] INFO: userDao: 有人访问了
[2022-10-31 07:16:26.225345] INFO: userDao: 有人访问了
[2022-10-31 07:16:43.901115] INFO: userDao: 有人访问了
[2022-10-31 07:17:29.205161] INFO: userDao: 有人访问了
[2022-10-31 07:18:26.766692] INFO: userDao: 有人访问了
[2022-10-31 07:20:19.209035] INFO: userDao: 有人访问了
[2022-10-31 07:20:22.203487] INFO: userDao: 有人访问了
[2022-10-31 07:20:42.945365] INFO: userDao: 有人访问了
[2022-10-31 07:21:34.407597] INFO: userDao: 有人访问了登录页面
[2022-10-31 07:24:33.429249] INFO: userDao: 有人访问了登录页面
[2022-10-31 07:25:55.639755] INFO: userDao: 有人访问了登录页面
[2022-10-31 07:31:06.025078] INFO: userDao: 有人访问了登录页面
[2022-11-01 08:36:03.172527] INFO: userDao: 有人访问了登录页面
[2022-11-01 11:18:49.265226] INFO: userDao: 有人访问了登录页面
[2022-11-01 16:06:59.643247] ERROR: userDao: 获取用户:1项目列表失败,Boolean value of this clause is not defined
[2022-11-01 16:11:44.431979] ERROR: userDao: 获取用户:1项目列表失败,Boolean value of this clause is not defined
[2022-11-01 16:13:26.868626] ERROR: userDao: 获取用户:1项目列表失败,Boolean value of this clause is not defined
[2022-11-01 16:17:03.538594] ERROR: userDao: 获取用户:1项目列表失败,Boolean value of this clause is not defined
[2022-11-01 16:18:43.667859] ERROR: userDao: 获取用户:1项目列表失败,Boolean value of this clause is not defined
[2022-11-01 16:23:36.663614] ERROR: userDao: 获取用户:1项目列表失败,type object 'Project' has no attribute 'owner'
[2022-11-01 17:03:37.450527] ERROR: userDao: 新增项目:first_project失败，Entity namespace for "project" has no property "name"
[2022-11-01 17:07:44.864902] ERROR: userDao: 新增项目:first_project失败，(_mysql_exceptions.OperationalError) (1054, "Unknown column 'project.name' in 'field list'")
[SQL: SELECT project.id AS project_id, project.name AS project_name, project.owner AS project_owner, project.created_at AS project_created_at, project.updated_at AS project_updated_at, project.deleted_at AS project_deleted_at, project.create_user AS project_create_user, project.update_user AS project_update_user, project.private AS project_private 
FROM project 
WHERE project.name = %s AND project.deleted_at IS NULL 
 LIMIT %s]
[parameters: ('first_project', 1)]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
[2022-11-01 17:09:24.443628] ERROR: userDao: 新增项目:first_project失败，commit() takes 1 positional argument but 2 were given
[2022-11-01 19:42:34.428978] INFO: userDao: 有人访问了登录页面
[2022-11-01 19:51:02.708247] INFO: userDao: 有人访问了登录页面
[2022-11-01 20:21:26.630043] ERROR: userDao: 新增项目:阿里百度联手项目失败，(_mysql_exceptions.OperationalError) (1267, "Illegal mix of collations (latin1_swedish_ci,IMPLICIT) and (utf8_general_ci,COERCIBLE) for operation '='")
[SQL: SELECT project.id AS project_id, project.name AS project_name, project.owner AS project_owner, project.created_at AS project_created_at, project.updated_at AS project_updated_at, project.deleted_at AS project_deleted_at, project.create_user AS project_create_user, project.update_user AS project_update_user, project.private AS project_private, project.img AS project_img 
FROM project 
WHERE project.name = %s AND project.deleted_at IS NULL 
 LIMIT %s]
[parameters: ('阿里百度联手项目', 1)]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
[2022-11-01 20:27:21.426255] ERROR: userDao: 新增项目:阿里百度联手项目失败，(_mysql_exceptions.OperationalError) (1267, "Illegal mix of collations (latin1_swedish_ci,IMPLICIT) and (utf8_general_ci,COERCIBLE) for operation '='")
[SQL: SELECT project.id AS project_id, project.name AS project_name, project.owner AS project_owner, project.created_at AS project_created_at, project.updated_at AS project_updated_at, project.deleted_at AS project_deleted_at, project.create_user AS project_create_user, project.update_user AS project_update_user, project.private AS project_private, project.img AS project_img 
FROM project 
WHERE project.name = %s AND project.deleted_at IS NULL 
 LIMIT %s]
[parameters: ('阿里百度联手项目', 1)]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
[2022-11-01 20:27:24.700421] ERROR: userDao: 新增项目:阿里百度联手项目失败，(_mysql_exceptions.OperationalError) (1267, "Illegal mix of collations (latin1_swedish_ci,IMPLICIT) and (utf8_general_ci,COERCIBLE) for operation '='")
[SQL: SELECT project.id AS project_id, project.name AS project_name, project.owner AS project_owner, project.created_at AS project_created_at, project.updated_at AS project_updated_at, project.deleted_at AS project_deleted_at, project.create_user AS project_create_user, project.update_user AS project_update_user, project.private AS project_private, project.img AS project_img 
FROM project 
WHERE project.name = %s AND project.deleted_at IS NULL 
 LIMIT %s]
[parameters: ('阿里百度联手项目', 1)]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
[2022-11-01 20:27:43.954822] ERROR: userDao: 新增项目:阿里百度联手项目失败，(_mysql_exceptions.OperationalError) (1267, "Illegal mix of collations (latin1_swedish_ci,IMPLICIT) and (utf8_general_ci,COERCIBLE) for operation '='")
[SQL: SELECT project.id AS project_id, project.name AS project_name, project.owner AS project_owner, project.created_at AS project_created_at, project.updated_at AS project_updated_at, project.deleted_at AS project_deleted_at, project.create_user AS project_create_user, project.update_user AS project_update_user, project.private AS project_private, project.img AS project_img 
FROM project 
WHERE project.name = %s AND project.deleted_at IS NULL 
 LIMIT %s]
[parameters: ('阿里百度联手项目', 1)]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
[2022-11-01 20:30:36.358681] ERROR: userDao: 新增项目:阿里百度联手项目失败，(_mysql_exceptions.OperationalError) (1267, "Illegal mix of collations (latin1_swedish_ci,IMPLICIT) and (utf8_general_ci,COERCIBLE) for operation '='")
[SQL: SELECT project.id AS project_id, project.name AS project_name, project.owner AS project_owner, project.created_at AS project_created_at, project.updated_at AS project_updated_at, project.deleted_at AS project_deleted_at, project.create_user AS project_create_user, project.update_user AS project_update_user, project.private AS project_private, project.img AS project_img 
FROM project 
WHERE project.name = %s AND project.deleted_at IS NULL 
 LIMIT %s]
[parameters: ('阿里百度联手项目', 1)]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
[2022-11-01 20:30:50.446260] ERROR: userDao: 新增项目:阿里百度联手项目失败，(_mysql_exceptions.OperationalError) (1267, "Illegal mix of collations (latin1_swedish_ci,IMPLICIT) and (utf8_general_ci,COERCIBLE) for operation '='")
[SQL: SELECT project.id AS project_id, project.name AS project_name, project.owner AS project_owner, project.created_at AS project_created_at, project.updated_at AS project_updated_at, project.deleted_at AS project_deleted_at, project.create_user AS project_create_user, project.update_user AS project_update_user, project.private AS project_private, project.img AS project_img 
FROM project 
WHERE project.name = %s AND project.deleted_at IS NULL 
 LIMIT %s]
[parameters: ('阿里百度联手项目', 1)]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
[2022-11-01 21:34:36.731439] ERROR: userDao: 查询用户:1项目失败,(_mysql_exceptions.OperationalError) (2013, 'Lost connection to MySQL server during query')
[SQL: SELECT project_role.id AS project_role_id, project_role.user_id AS project_role_user_id, project_role.project_id AS project_role_project_id, project_role.project_role AS project_role_project_role, project_role.created_at AS project_role_created_at, project_role.updated_at AS project_role_updated_at, project_role.deleted_at AS project_role_deleted_at, project_role.create_user AS project_role_create_user, project_role.update_user AS project_role_update_user 
FROM project_role 
WHERE project_role.user_id = %s AND project_role.deleted_at IS NULL]
[parameters: (1,)]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
[2022-11-01 21:34:36.734455] ERROR: userDao: 获取用户:1项目列表失败,Can't reconnect until invalid transaction is rolled back. (Background on this error at: https://sqlalche.me/e/14/8s2b)
[2022-11-01 23:13:35.551558] INFO: userDao: 有人访问了登录页面
[2022-11-01 23:14:02.656030] ERROR: userDao: 新增项目:我的失败，(_mysql_exceptions.OperationalError) (1267, "Illegal mix of collations (latin1_swedish_ci,IMPLICIT) and (utf8_general_ci,COERCIBLE) for operation '='")
[SQL: SELECT project.id AS project_id, project.name AS project_name, project.owner AS project_owner, project.created_at AS project_created_at, project.updated_at AS project_updated_at, project.deleted_at AS project_deleted_at, project.create_user AS project_create_user, project.update_user AS project_update_user, project.private AS project_private, project.img AS project_img 
FROM project 
WHERE project.name = %s AND project.deleted_at IS NULL 
 LIMIT %s]
[parameters: ('我的', 1)]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
[2022-11-01 23:15:18.976537] ERROR: userDao: 新增项目:我的失败，(_mysql_exceptions.OperationalError) (1267, "Illegal mix of collations (latin1_swedish_ci,IMPLICIT) and (utf8_general_ci,COERCIBLE) for operation '='")
[SQL: SELECT project.id AS project_id, project.name AS project_name, project.owner AS project_owner, project.created_at AS project_created_at, project.updated_at AS project_updated_at, project.deleted_at AS project_deleted_at, project.create_user AS project_create_user, project.update_user AS project_update_user, project.private AS project_private, project.img AS project_img 
FROM project 
WHERE project.name = %s AND project.deleted_at IS NULL 
 LIMIT %s]
[parameters: ('我的', 1)]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
[2022-11-01 23:15:41.209111] ERROR: userDao: 新增项目:我的失败，(_mysql_exceptions.OperationalError) (1267, "Illegal mix of collations (latin1_swedish_ci,IMPLICIT) and (utf8_general_ci,COERCIBLE) for operation '='")
[SQL: SELECT project.id AS project_id, project.name AS project_name, project.owner AS project_owner, project.created_at AS project_created_at, project.updated_at AS project_updated_at, project.deleted_at AS project_deleted_at, project.create_user AS project_create_user, project.update_user AS project_update_user, project.private AS project_private, project.img AS project_img 
FROM project 
WHERE project.name = %s AND project.deleted_at IS NULL 
 LIMIT %s]
[parameters: ('我的', 1)]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
[2022-11-01 23:17:57.136381] ERROR: userDao: 新增项目:我的失败，(_mysql_exceptions.OperationalError) (1267, "Illegal mix of collations (latin1_swedish_ci,IMPLICIT) and (utf8_general_ci,COERCIBLE) for operation '='")
[SQL: SELECT project.id AS project_id, project.name AS project_name, project.owner AS project_owner, project.created_at AS project_created_at, project.updated_at AS project_updated_at, project.deleted_at AS project_deleted_at, project.create_user AS project_create_user, project.update_user AS project_update_user, project.private AS project_private, project.img AS project_img 
FROM project 
WHERE project.name = %s AND project.deleted_at IS NULL 
 LIMIT %s]
[parameters: ('我的', 1)]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
[2022-11-01 23:20:15.023213] ERROR: userDao: 新增项目:我的失败，(_mysql_exceptions.OperationalError) (2013, 'Lost connection to MySQL server during query')
[SQL: SELECT project.id AS project_id, project.name AS project_name, project.owner AS project_owner, project.created_at AS project_created_at, project.updated_at AS project_updated_at, project.deleted_at AS project_deleted_at, project.create_user AS project_create_user, project.update_user AS project_update_user, project.private AS project_private, project.img AS project_img 
FROM project 
WHERE project.name = %s AND project.deleted_at IS NULL 
 LIMIT %s]
[parameters: ('我的', 1)]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
[2022-11-01 23:20:29.005955] ERROR: userDao: 新增项目:我的失败，(_mysql_exceptions.OperationalError) (1267, "Illegal mix of collations (latin1_swedish_ci,IMPLICIT) and (utf8_general_ci,COERCIBLE) for operation '='")
[SQL: SELECT project.id AS project_id, project.name AS project_name, project.owner AS project_owner, project.created_at AS project_created_at, project.updated_at AS project_updated_at, project.deleted_at AS project_deleted_at, project.create_user AS project_create_user, project.update_user AS project_update_user, project.private AS project_private, project.img AS project_img 
FROM project 
WHERE project.name = %s AND project.deleted_at IS NULL 
 LIMIT %s]
[parameters: ('我的', 1)]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
[2022-11-01 23:24:25.478480] ERROR: userDao: 新增项目:我的失败，(_mysql_exceptions.OperationalError) (1267, "Illegal mix of collations (latin1_swedish_ci,IMPLICIT) and (utf8_general_ci,COERCIBLE) for operation '='")
[SQL: SELECT project.id AS project_id, project.name AS project_name, project.owner AS project_owner, project.created_at AS project_created_at, project.updated_at AS project_updated_at, project.deleted_at AS project_deleted_at, project.create_user AS project_create_user, project.update_user AS project_update_user, project.private AS project_private, project.img AS project_img 
FROM project 
WHERE project.name = %s AND project.deleted_at IS NULL 
 LIMIT %s]
[parameters: ('我的', 1)]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
[2022-11-01 23:25:47.210361] ERROR: userDao: 新增项目:我的失败，(_mysql_exceptions.OperationalError) (2013, 'Lost connection to MySQL server during query')
[SQL: SELECT project.id AS project_id, project.name AS project_name, project.owner AS project_owner, project.created_at AS project_created_at, project.updated_at AS project_updated_at, project.deleted_at AS project_deleted_at, project.create_user AS project_create_user, project.update_user AS project_update_user, project.private AS project_private, project.img AS project_img 
FROM project 
WHERE project.name = %s AND project.deleted_at IS NULL 
 LIMIT %s]
[parameters: ('我的', 1)]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
[2022-11-01 23:25:51.731170] ERROR: userDao: 新增项目:我的失败，(_mysql_exceptions.OperationalError) (1267, "Illegal mix of collations (latin1_swedish_ci,IMPLICIT) and (utf8_general_ci,COERCIBLE) for operation '='")
[SQL: SELECT project.id AS project_id, project.name AS project_name, project.owner AS project_owner, project.created_at AS project_created_at, project.updated_at AS project_updated_at, project.deleted_at AS project_deleted_at, project.create_user AS project_create_user, project.update_user AS project_update_user, project.private AS project_private, project.img AS project_img 
FROM project 
WHERE project.name = %s AND project.deleted_at IS NULL 
 LIMIT %s]
[parameters: ('我的', 1)]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
[2022-11-01 23:25:52.977638] ERROR: userDao: 新增项目:我的失败，(_mysql_exceptions.OperationalError) (1267, "Illegal mix of collations (latin1_swedish_ci,IMPLICIT) and (utf8_general_ci,COERCIBLE) for operation '='")
[SQL: SELECT project.id AS project_id, project.name AS project_name, project.owner AS project_owner, project.created_at AS project_created_at, project.updated_at AS project_updated_at, project.deleted_at AS project_deleted_at, project.create_user AS project_create_user, project.update_user AS project_update_user, project.private AS project_private, project.img AS project_img 
FROM project 
WHERE project.name = %s AND project.deleted_at IS NULL 
 LIMIT %s]
[parameters: ('我的', 1)]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
[2022-11-02 09:59:56.803092] ERROR: userDao: 新增项目:我的失败，(_mysql_exceptions.OperationalError) (1267, "Illegal mix of collations (latin1_swedish_ci,IMPLICIT) and (utf8_general_ci,COERCIBLE) for operation '='")
[SQL: SELECT project.id AS project_id, project.name AS project_name, project.owner AS project_owner, project.created_at AS project_created_at, project.updated_at AS project_updated_at, project.deleted_at AS project_deleted_at, project.create_user AS project_create_user, project.update_user AS project_update_user, project.private AS project_private, project.img AS project_img 
FROM project 
WHERE project.name = %s AND project.deleted_at IS NULL 
 LIMIT %s]
[parameters: ('我的', 1)]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
[2022-11-02 11:45:36.276740] INFO: userDao: 有人访问了登录页面
[2022-11-02 14:07:31.327287] ERROR: userDao: 新增项目:tes123t33543f33s23失败，(_mysql_exceptions.DataError) (1406, "Data too long for column 'name' at row 1")
[SQL: INSERT INTO project (name, owner, created_at, updated_at, deleted_at, create_user, update_user, private, img, description) VALUES (%s, %s, %s, %s, %s, %s, %s, %s, %s, %s)]
[parameters: ('tes123t33543f33s23', 1, datetime.datetime(2022, 11, 2, 14, 7, 31, 313200), datetime.datetime(2022, 11, 2, 14, 7, 31, 313200), None, 1, 1, 1, 'https://t7.baidu.com/it/u=2359570649,2574326109&fm=193&f=GIF', '')]
(Background on this error at: https://sqlalche.me/e/14/9h9h)
[2022-11-02 14:08:27.777446] ERROR: userDao: 新增项目:test234失败，(_mysql_exceptions.OperationalError) (1366, "Incorrect integer value: 'on' for column 'private' at row 1")
[SQL: INSERT INTO project (name, owner, created_at, updated_at, deleted_at, create_user, update_user, private, img, description) VALUES (%s, %s, %s, %s, %s, %s, %s, %s, %s, %s)]
[parameters: ('test234', '0', datetime.datetime(2022, 11, 2, 14, 8, 27, 766882), datetime.datetime(2022, 11, 2, 14, 8, 27, 766882), None, '0', '0', 'on', 'https://t7.baidu.com/it/u=836228418,3299829885&fm=193&f=GIF', '123123')]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
[2022-11-02 14:09:30.413289] ERROR: userDao: 新增项目:test234失败，(_mysql_exceptions.OperationalError) (1366, "Incorrect integer value: 'on' for column 'private' at row 1")
[SQL: INSERT INTO project (name, owner, created_at, updated_at, deleted_at, create_user, update_user, private, img, description) VALUES (%s, %s, %s, %s, %s, %s, %s, %s, %s, %s)]
[parameters: ('test234', '0', datetime.datetime(2022, 11, 2, 14, 9, 30, 386096), datetime.datetime(2022, 11, 2, 14, 9, 30, 386096), None, '0', '0', 'on', 'https://t7.baidu.com/it/u=2177360828,736658999&fm=193&f=GIF', '123123')]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
[2022-11-02 14:14:15.013603] ERROR: userDao: 新增项目:test234失败，(_mysql_exceptions.OperationalError) (1366, "Incorrect integer value: 'on' for column 'private' at row 1")
[SQL: INSERT INTO project (name, owner, created_at, updated_at, deleted_at, create_user, update_user, private, img, description) VALUES (%s, %s, %s, %s, %s, %s, %s, %s, %s, %s)]
[parameters: ('test234', '0', datetime.datetime(2022, 11, 2, 14, 14, 14, 998143), datetime.datetime(2022, 11, 2, 14, 14, 14, 998143), None, '0', '0', 'on', 'https://t7.baidu.com/it/u=3985596329,30052956&fm=193&f=GIF', '123123')]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
[2022-11-02 14:21:19.930676] ERROR: userDao: 新增项目:123123失败，(_mysql_exceptions.OperationalError) (1366, "Incorrect integer value: 'on' for column 'private' at row 1")
[SQL: INSERT INTO project (name, owner, created_at, updated_at, deleted_at, create_user, update_user, private, img, description) VALUES (%s, %s, %s, %s, %s, %s, %s, %s, %s, %s)]
[parameters: ('123123', '0', datetime.datetime(2022, 11, 2, 14, 21, 19, 918269), datetime.datetime(2022, 11, 2, 14, 21, 19, 918269), None, '0', '0', 'on', 'https://t7.baidu.com/it/u=1811223786,2017754440&fm=193&f=GIF', '123123')]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
[2022-11-02 14:40:26.138482] INFO: userDao: 有人访问了登录页面
[2022-11-02 14:47:18.195714] INFO: userDao: 有人访问了登录页面
[2022-11-02 18:12:42.517821] INFO: userDao: 有人访问了登录页面
[2022-11-03 12:01:09.671684] INFO: userDao: 有人访问了登录页面
[2022-11-05 16:29:40.449939] INFO: userDao: 有人访问了登录页面
[2022-11-05 22:50:59.225219] INFO: userDao: 有人访问了登录页面
[2022-11-05 23:03:33.016207] ERROR: userDao: 查询项目:24用户失败,get() got an unexpected keyword argument 'id'
[2022-11-05 23:04:18.520130] ERROR: userDao: 查询项目:24用户失败,get() got an unexpected keyword argument 'id'
[2022-11-05 23:12:22.876143] ERROR: userDao: 查询项目:24用户失败,get() got an unexpected keyword argument 'id'
[2022-11-06 14:07:46.644329] INFO: userDao: 有人访问了登录页面
[2022-11-07 12:21:43.651857] INFO: userDao: 有人访问了登录页面
[2022-11-07 18:43:38.945179] INFO: userDao: 有人访问了登录页面
[2022-11-08 18:54:57.983959] INFO: userDao: 有人访问了登录页面
[2022-11-09 08:19:42.795832] INFO: userDao: 有人访问了登录页面
[2022-11-09 09:38:12.583614] ERROR: userDao: 获取用户:1项目列表失败,(_mysql_exceptions.OperationalError) (1054, "Unknown column 'project.app' in 'field list'")
[SQL: SELECT count(*) AS count_1 
FROM (SELECT project.id AS project_id, project.name AS project_name, project.owner AS project_owner, project.app AS project_app, project.created_at AS project_created_at, project.updated_at AS project_updated_at, project.deleted_at AS project_deleted_at, project.create_user AS project_create_user, project.update_user AS project_update_user, project.private AS project_private, project.img AS project_img, project.description AS project_description 
FROM project 
WHERE project.deleted_at IS NULL AND (project.owner = %s OR project.private = %s)) AS anon_1]
[parameters: (1, 0)]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
[2022-11-09 09:38:14.642148] ERROR: userDao: 获取用户:1项目列表失败,(_mysql_exceptions.OperationalError) (1054, "Unknown column 'project.app' in 'field list'")
[SQL: SELECT count(*) AS count_1 
FROM (SELECT project.id AS project_id, project.name AS project_name, project.owner AS project_owner, project.app AS project_app, project.created_at AS project_created_at, project.updated_at AS project_updated_at, project.deleted_at AS project_deleted_at, project.create_user AS project_create_user, project.update_user AS project_update_user, project.private AS project_private, project.img AS project_img, project.description AS project_description 
FROM project 
WHERE project.deleted_at IS NULL AND (project.owner = %s OR project.private = %s)) AS anon_1]
[parameters: (1, 0)]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
[2022-11-09 09:38:21.182605] ERROR: userDao: 获取用户:1项目列表失败,(_mysql_exceptions.OperationalError) (1054, "Unknown column 'project.app' in 'field list'")
[SQL: SELECT count(*) AS count_1 
FROM (SELECT project.id AS project_id, project.name AS project_name, project.owner AS project_owner, project.app AS project_app, project.created_at AS project_created_at, project.updated_at AS project_updated_at, project.deleted_at AS project_deleted_at, project.create_user AS project_create_user, project.update_user AS project_update_user, project.private AS project_private, project.img AS project_img, project.description AS project_description 
FROM project 
WHERE project.deleted_at IS NULL AND (project.owner = %s OR project.private = %s)) AS anon_1]
[parameters: (1, 0)]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
[2022-11-09 09:38:43.752382] ERROR: userDao: 获取用户:1项目列表失败,(_mysql_exceptions.OperationalError) (1054, "Unknown column 'project.app' in 'field list'")
[SQL: SELECT count(*) AS count_1 
FROM (SELECT project.id AS project_id, project.name AS project_name, project.owner AS project_owner, project.app AS project_app, project.created_at AS project_created_at, project.updated_at AS project_updated_at, project.deleted_at AS project_deleted_at, project.create_user AS project_create_user, project.update_user AS project_update_user, project.private AS project_private, project.img AS project_img, project.description AS project_description 
FROM project 
WHERE project.deleted_at IS NULL AND (project.owner = %s OR project.private = %s)) AS anon_1]
[parameters: (1, 0)]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
[2022-11-09 09:39:58.329965] ERROR: userDao: 获取用户:1项目列表失败,(_mysql_exceptions.OperationalError) (1054, "Unknown column 'project.app' in 'field list'")
[SQL: SELECT count(*) AS count_1 
FROM (SELECT project.id AS project_id, project.name AS project_name, project.owner AS project_owner, project.app AS project_app, project.created_at AS project_created_at, project.updated_at AS project_updated_at, project.deleted_at AS project_deleted_at, project.create_user AS project_create_user, project.update_user AS project_update_user, project.private AS project_private, project.img AS project_img, project.description AS project_description 
FROM project 
WHERE project.deleted_at IS NULL AND (project.owner = %s OR project.private = %s)) AS anon_1]
[parameters: (1, 0)]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
[2022-11-09 09:41:21.570065] ERROR: userDao: 执行用例失败: the JSON object must be str, bytes or bytearray, not 'NoneType'
[2022-11-09 09:42:46.685804] ERROR: userDao: 执行用例失败: the JSON object must be str, bytes or bytearray, not 'NoneType'
[2022-11-09 09:43:02.300778] ERROR: userDao: 执行用例失败: the JSON object must be str, bytes or bytearray, not 'NoneType'
[2022-11-09 09:45:49.150794] ERROR: userDao: 执行用例失败: the JSON object must be str, bytes or bytearray, not 'NoneType'
[2022-11-09 09:45:49.159231] ERROR: userDao: 执行用例失败: 'NoneType' object has no attribute 'request_header'
[2022-11-09 09:45:49.169797] ERROR: userDao: 执行用例失败: the JSON object must be str, bytes or bytearray, not 'NoneType'
[2022-11-09 14:58:15.971152] INFO: userDao: 有人访问了登录页面
[2022-11-09 17:39:54.678744] INFO: userDao: 有人访问了登录页面
[2022-11-09 17:55:11.344069] ERROR: userDao: 连接数据库82.157.253.2373失败: (2003, "Can't connect to MySQL server on '82.157.253.2373' ([Errno 11001] getaddrinfo failed)")
[2022-11-10 09:20:45.382811] INFO: userDao: 有人访问了登录页面
[2022-11-10 14:45:40.341463] INFO: userDao: 有人访问了登录页面
[2022-11-11 09:30:40.664743] INFO: userDao: 有人访问了登录页面
[2022-11-14 09:44:53.977905] INFO: userDao: 有人访问了登录页面
[2022-11-14 09:45:40.572070] ERROR: userDao: 连接redis数据库192.168.4.16失败: 'Blueprint' object has no attribute 'Redis'
[2022-11-14 09:46:02.689715] ERROR: userDao: 连接redis数据库192.168.4.16失败: 'Blueprint' object has no attribute 'Redis'
[2022-11-14 09:46:35.675001] ERROR: userDao: 连接redis数据库192.168.4.16失败: 'Blueprint' object has no attribute 'Redis'
[2022-11-14 09:47:05.328602] ERROR: userDao: 连接redis数据库192.168.4.16失败: 'Blueprint' object has no attribute 'Redis'
[2022-11-14 09:48:48.910724] ERROR: userDao: 连接redis数据库192.168.4.16失败: 'Blueprint' object has no attribute 'Redis'
[2022-11-14 09:53:25.811692] ERROR: userDao: 连接redis数据库192.168.4.16失败: 'Blueprint' object has no attribute 'Redis'
[2022-11-14 09:53:45.558564] ERROR: userDao: 连接redis数据库192.168.4.16失败: 'Blueprint' object has no attribute 'Redis'
[2022-11-14 10:09:13.421764] ERROR: userDao: 连接数据库192.168.4.161失败: 'Redis' object has no attribute 'Redis'
[2022-11-14 10:09:28.118717] ERROR: userDao: 连接数据库192.168.4.16失败: 'Redis' object has no attribute 'Redis'
[2022-11-14 10:13:47.389453] ERROR: userDao: 连接数据库192.168.4.16失败: 'Redis' object has no attribute 'Redis'
[2022-11-17 15:40:27.097961] INFO: userDao: 有人访问了登录页面
[2022-11-17 19:39:29.714453] INFO: userDao: 有人访问了登录页面
[2022-11-18 17:11:13.748208] INFO: userDao: 有人访问了登录页面
[2022-11-18 18:32:37.623071] INFO: userDao: 有人访问了登录页面
[2022-11-19 15:01:20.903449] INFO: userDao: 有人访问了登录页面
[2022-11-21 09:19:43.987270] INFO: userDao: 有人访问了登录页面
[2022-11-21 12:04:11.512760] ERROR: userDao: 执行用例失败: 'NoneType' object has no attribute 'request_header'
[2022-11-21 12:04:11.528298] ERROR: userDao: 执行用例失败: 'NoneType' object has no attribute 'request_header'
[2022-11-21 12:04:11.542699] ERROR: userDao: 执行用例失败: 'NoneType' object has no attribute 'request_header'
[2022-11-21 12:04:11.554744] ERROR: userDao: 执行用例失败: 'NoneType' object has no attribute 'request_header'
[2022-11-21 12:04:11.564804] ERROR: userDao: 执行用例失败: 'NoneType' object has no attribute 'request_header'
[2022-11-21 14:16:15.619980] INFO: userDao: 有人访问了登录页面
[2022-11-22 09:25:39.685780] INFO: userDao: 有人访问了登录页面
[2022-11-22 16:23:25.094321] INFO: userDao: 有人访问了登录页面
[2022-11-22 17:45:54.704366] ERROR: userDao: 新增项目:民生银行联邦平台失败，(_mysql_exceptions.OperationalError) (1267, "Illegal mix of collations (latin1_swedish_ci,IMPLICIT) and (utf8_general_ci,COERCIBLE) for operation '='")
[SQL: SELECT project.id AS project_id, project.name AS project_name, project.owner AS project_owner, project.app AS project_app, project.created_at AS project_created_at, project.updated_at AS project_updated_at, project.deleted_at AS project_deleted_at, project.create_user AS project_create_user, project.update_user AS project_update_user, project.private AS project_private, project.img AS project_img, project.description AS project_description, project.`accessToken` AS `project_accessToken` 
FROM project 
WHERE project.name = %s AND project.deleted_at IS NULL 
 LIMIT %s]
[parameters: ('民生银行联邦平台', 1)]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
[2022-11-24 07:55:51.253570] INFO: userDao: 有人访问了登录页面
[2022-11-27 22:07:29.738359] INFO: userDao: 有人访问了登录页面
[2022-11-28 09:41:47.034449] INFO: userDao: 有人访问了登录页面
[2022-11-28 09:42:12.879721] ERROR: userDao: 用户hanfengju登录失败:(_mysql_exceptions.OperationalError) (2013, 'Lost connection to MySQL server during query')
[SQL: SELECT user.id AS user_id, user.username AS user_username, user.name AS user_name, user.password AS user_password, user.email AS user_email, user.`role` AS user_role, user.created_at AS user_created_at, user.updated_at AS user_updated_at, user.deleted_at AS user_deleted_at, user.last_login_at AS user_last_login_at 
FROM user 
WHERE user.username = %s AND user.password = %s AND user.deleted_at IS NULL 
 LIMIT %s]
[parameters: ('hanfengju', '3032cf02526b924f057454fc31468f8e', 1)]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
[2022-11-28 15:19:37.729956] INFO: userDao: 有人访问了登录页面
[2022-11-29 10:12:19.687880] INFO: userDao: 有人访问了登录页面
[2022-11-29 14:45:35.770284] INFO: userDao: 有人访问了登录页面
[2022-11-29 17:32:22.078597] INFO: userDao: 有人访问了登录页面
[2022-11-29 17:32:39.690571] INFO: userDao: 有人访问了登录页面
[2022-11-29 19:24:05.055954] INFO: userDao: 有人访问了登录页面
[2022-11-30 17:12:43.379934] INFO: userDao: 有人访问了登录页面
[2022-11-30 20:25:43.804872] INFO: userDao: 有人访问了登录页面
